seed/utils/meters.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, property_id, org_id, excluded_meter_ids, scenario_ids=None):
         self._cache_factors = None
         self._cache_org_country = None
 
+        scenario_ids = scenario_ids if scenario_ids is not None else []
```

Comment:
I think we could also set the default for `scenario_ids` as `[]` when defining the parameters.

Reply:
I used this pattern because python's default parameters are only evaluated once when the function is defined, so dicts and lists can cause some surprising results if modified. Check this out here: https://docs.python-guide.org/writing/gotchas/

Reply:
Ah ok - didn't realize this could have been an issue! 👍